extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -262,6 +263,33 @@ export class AmpStoryPage extends AMP.BaseElement {
     );
   }
 
+  /**
+   * @private
+   * @return {Element}
+   */
+  maybeConvertCtaLayerToPageOutlink_() {
+    const ctaLayerEl = this.element.querySelector('amp-story-cta-layer');
+    if (!ctaLayerEl) {
+      return;
+    }
+
+    const anchorSet = ctaLayerEl.querySelectorAll('a');
+    if (anchorSet.length !== 1 || !anchorSet[0].getAttribute('href')) {
+      return;
+    }
+
+    this.mutateElement(() => {
```

Comment:
`buildCallback` already runs in a `mutateElement` context, so in this specific case you don't need to wrap your statements in this callback.

The cool thing is, it'll happen synchronously, and you should be able to remove your selector update in `installPageAttachmentExtension_`.